module/solutions/helper.js

```diff
@@ -2732,6 +2732,20 @@ module.exports = class SolutionsHelper {
 				}
 
 				if (targetedSolutions.success) {
+					if (!(targetedSolutions.data.data.length > 0) && currentScopeOnly) {
```

Comment:
@Prajwal17Tunerlabs can we add a comment explaining why we have to make this change

Reply:
i have added comments